samples/src/main.c

```diff
@@ -3,27 +3,197 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-#include <zephyr/kernel.h>
 #include <zephyr/device.h>
-#include <zephyr/drivers/sensor.h>
+#include <zephyr/devicetree.h>
+#include <zephyr/drivers/display.h>
+#include <zephyr/drivers/gpio.h>
+#include <zephyr/drivers/pwm.h>
+#include <lvgl.h>
+#include <stdio.h>
+#include <string.h>
+#include <zephyr/kernel.h>
+#include <lvgl_input_device.h>
```

Comment:
On this example we don't have any input device. I think we didn't need to include `lvgl_input_device.h`

Reply:
Indeed we don't need input device, all related code will be deleted in next commit

Reply:
Resolved in [1107a99](https://github.com/catie-aq/zephyr_ilitek-ili9163c/pull/3/commits/1107a9950e66be1f2fa57e347fc6bf2b0cb3b9f8).